gettext_with_fallback.py

```diff
@@ -0,0 +1,8 @@
+from flask_babel import gettext
+from website.flask_helpers import gettext_with_fallback
+
+
+# Explicitly substitute the flask_babel function gettext with our own implementation that
+# adds a fallback language. Note that we need to monkey-patch instead of use the gettext_with_fallback
+# directly because the Babel extract function works if we use gettext with literal strings.
```

Comment:
I'm not entirely following what this is trying to say.

This file seems to be a re-export of a symbol elsewhere, so that we can do:

```py
from gettext_with_fallback import gettext
```

Instead of doing:

```py
from website.flask_helpers import gettext_with_fallback as gettext
```

Is that right?

Reply:
Thanks for the feedback, I should update the comment then! I wrote it down as a reminder that I cannot use directly in the python `gettext_with_fallback('msg')` instead of `gettext('msg')` because the extract will not work (although this would be ideal because we decrease the chance of someone using the gettext without a fallback in the future). We have a hack with the client messages to workaround that. I will think of a better way to phrase it.

About the import, you are totally right! The idea was to have a single line of code and I did not realize I can to the `import` and `as` on the same line. So, yes, this is exactly my intention.